### PR TITLE
Re-create Awesome Books  (ES6)

### PR DESCRIPTION
**Pull request by Mar12358**
### Added main files:
- index.js
- index.css
- index.html
### Added .js modules
- books.js
- bookControllers.js
- div.js
- emptyValue.js
### Useage ES6 syntax
- Code Divided into modules
- Usage of let and const
- Usage of arrow functions
### Display current date
- install using "npm install --save luxon"
- Usage of Luxon library imported as module
### Documentation
- Add README file.
- See a [Live Demo with gh-pages](https://mar12358.github.io/ES6-Awesome-Books/)

### CODE REVIEW

**Review by LionRouge1 — COMMENTED** [review]
Hi @Mar12358, 

done

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional REAMDE file. Kindly use the provided README template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const data = this.getLocalStorage();
+    bookDetails.id = data.length + 1;
+    data.push(bookDetails);
+    this.setLocalStorage(data);
+  }
+
+  removeBook(id) {
+    const data = this.getLocalStorage().filter((e) => e.id !== id);
+    this.setLocalStorage(data);
+  }
```
Comment (thecodechaser) [inline modules/book.js]: - [ ] Please change these functions to arrow functions to follow the ES6 syntax as requested in the project requirements.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Mar12358👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Highly organized code. ✔️
- No linter errors. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @Mar12358, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- JS code organized in modules :heavy_check_mark: 
- Luxon used to display date :heavy_check_mark: 
- ES6 features utilized :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**